mlrun/serving/routers.py

```diff
@@ -641,3 +656,41 @@ def validate(self, request):
             if not isinstance(request["inputs"], list):
                 raise Exception('Expected "inputs" to be a list')
         return request
+
+
+def _init_endpoint_record(graph_server, voting_ensemble: VotingEnsemble):
+    logger.info("Initializing endpoint records")
+
+    try:
+        project, uri, tag, hash_key = parse_versioned_object_uri(
+            graph_server.function_uri
+        )
+
+        if voting_ensemble.version:
+            versioned_model_name = f"{voting_ensemble.name}:{voting_ensemble.version}"
+        else:
+            versioned_model_name = f"{voting_ensemble.name}:latest"
+
+        model_endpoint = ModelEndpoint(
+            metadata=ModelEndpointMetadata(project=project),
+            spec=ModelEndpointSpec(
+                function_uri=graph_server.function_uri,
+                model=versioned_model_name,
+                model_class=voting_ensemble.__class__.__name__,
+                stream_path=config.model_endpoint_monitoring.store_prefixes.default.format(
+                    project=project, kind="stream"
+                ),
+                active=True,
+            ),
+            status=ModelEndpointStatus(),
+        )
+
+        db = mlrun.get_run_db()
+
+        db.create_or_patch_model_endpoint(
+            project=project,
+            endpoint_id=model_endpoint.metadata.uid,
+            model_endpoint=model_endpoint,
+        )
+    except Exception as e:
+        logger.error("Failed to create endpoint record", exc=e)
```

Comment:
* Error is for catastrophe, if you ignore it, it surely not
* endpoint is very generic and it's not obvious that you meaning model endpoint
```suggestion
        logger.warning("Failed creating model endpoint record", exc=exc, traceback=traceback.format_exc())
```